<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:white;
}

#centralwidget{
background-color:white;
}


#page {
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="styleSheet">
       <string notr="true">#headerLine1 {
color: #10576D;
}

#headerLine2{
color:#10576D;
}

</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inside_experiment"/>
      <widget class="QWidget" name="outside_experiment">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="outside_experiment_header_layout" stretch="1,1,2,1">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="historyButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>histórico</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images/resources.qrc">
              <normaloff>:/lightHistory.png</normaloff>:/lightHistory.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="experimentButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>experimento</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images/resources.qrc">
              <normaloff>:/lightExperiment.png</normaloff>:/lightExperiment.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="configurationButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>configurações</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/lightConfiguration.png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="headerLine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>10</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#headerLine {
color: #10576D;
}</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="outside_experiment_stack">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="history_page">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>120</y>
              <width>591</width>
              <height>141</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Página do histórico em construção.</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="configuration_page">
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>130</y>
              <width>701</width>
              <height>141</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Página das configurações em construção.</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="experiment_page">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Etapa 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Etapa 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="etapas_stack">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_3"/>
              <widget class="QWidget" name="page_4">
               <widget class="QLabel" name="label_3">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>60</y>
                  <width>591</width>
                  <height>141</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>26</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Página etapas em construção.</string>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="after_experiment"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
